### PR TITLE
Update Docker log documentation

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
The new agent is coming and contains a lot of new features.
We also leverage the new tab design to improve the display of the information

### Motivation
New agent release

### Preview link
https://docs-staging.datadoghq.com/nils/docker-log-update/logs/log_collection/docker/?tab=containerinstallation

### CODE REVIEW

**Comment by NBParis** [comment]
@cswatt Did a couple modifications/additions, feel free to review them and share your thoughts

File: content/logs/log_collection/docker.md [hunk 1]
```diff
@@ -27,118 +27,122 @@ Two installations are possible:
...
+{{< /tabs >}}
 
-For more examples of configuration files or Agent capabilities (such as filtering, redacting, multiline, ...) read [the advanced log collection functions][7].
+### Activate Log Integrations
 
-#### Option 2: Autodiscovery
+The second step is to use Autodiscovery to customize the `source` and `service` value. This allows Datadog to identify the log source for each container.
 
-Since version 6.2 of the Datadog Agent, you can configure log collection directly in the container labels.
+Since version 6.2 of the Datadog Agent, you can configure log collection directly in the container labels.
```
Comment (ajacquemot) [inline content/logs/log_collection/docker.md]: Pod annotations work only since `6.5`

**Comment by ajacquemot** [comment]
LGTM

File: content/logs/log_collection/docker.md (hunk 1 above)
Comment (NBParis) [reply content/logs/log_collection/docker.md]: Yeah we should remove reference to specific version (container labels do work since 6.2 though)

**Pull request merged**